sdk/opentelemetry-sdk.gemspec

```diff
@@ -38,7 +38,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'bundler', '>= 1.17'
   spec.add_development_dependency 'faraday', '~> 0.13'
   spec.add_development_dependency 'minitest', '~> 5.0'
-  spec.add_development_dependency 'opentelemetry-exporter-zipkin', '~> 0.19.0'
+  spec.add_development_dependency 'opentelemetry-exporter-zipkin', '~> 0.19'
```

Comment:
I don't think there was any real intention to this patch version constraint, as it was just bumped from 0.18.0 => 0.19.0 https://github.com/open-telemetry/opentelemetry-ruby/blame/85e99f9e3e1bce90b23c5647f1bcf2008654c3bc/sdk/opentelemetry-sdk.gemspec#L41